src/app/_components/Card/CardSpeсialist/AddressesList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropType from 'prop-types';
+import { HospitalSVG } from '@icons/index';
+import { cn } from '@/utils/cn';
+import { borderStyle } from '@/app/_components/Card/CardSpeсialist/config';
+
+export function AddressesList({ addresses, className, showIcon = false }) {
```

Comment:
Consider using default props here and in other components
```js
AddressesList.defaultProps = {
  showIcon: false,
}

Reply:
Thanks for comment! Fixed it.

Reply:
Guys, there's a notification in the console that default props will be removed in the next major release. Are we going to go this way anyway or let's go back to the default props as shown on line 7?

![image](https://github.com/keenethics/mayak/assets/103147058/d78f3fa8-abb6-461b-8293-cf18da48dcdd)

Reply:
In this case, you should return it as it was.
@BigTako Could you double-check this?

Reply:
I see such warning message in console too. Sorry, didn't notice it at that moment, will remove it in next commit.